lexicon/conversion.py

```diff
@@ -42,12 +42,26 @@ def run(self):
 
 
 class FilterLexiconByWordListJob(Job):
```

Comment:
Adding a new parameter to `__init__` changes the hashes of all existing jobs, even if the old behavior is the same. To prevent this, there is an option called `__sis_hash_exclude__`

```suggestion
class FilterLexiconByWordListJob(Job):
    __sis_hash_exclude__ = {"check_synt_tok" : False}
```

I request changes w.r.t. this comment.